### PR TITLE
feat: add empty additional rule card

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
Just base work

### CODE REVIEW

File: apps/web/src/services/asset-inventory/components/CollectorAdditionalRule.vue [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+const collectorFormState = collectorFormStore.state;
+
+const state = reactive({
+    orderedCardData: computed<AdditionalRule[]>(() => {
+        const data = collectorFormState.additionalRules;
+        return data.sort((a, b) => a.order - b.order);
+    }),
+});
+
+const changeOrder = (targetData, clickedData, tempOrder) => {
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/components/CollectorAdditionalRule.vue:24]: Type Annotation~~

**Review by piggggggggy — APPROVED** [review]
훌륭합니다

**Review by skdud4659 — APPROVED** [review]
L
G
T
M

File: apps/web/src/services/asset-inventory/components/CollectorAdditionalRule.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/components/CollectorAdditionalRule.vue:24]: 복붙이라... 곧 새로 업데이트될 예정입니다!

**Pull request merged**